components/viz/common/quads/draw_quad_perftest.cc

```diff
@@ -71,8 +71,8 @@ class DrawQuadPerfTest : public testing::Test {
   }
 
   void CleanUpRenderPass() {
-    render_pass_.reset();
```

Comment:
Why this change? Is there any equivalent upstream change? If we're going to make it, even though it's small, it's in upstream code so we should wrap it with BUILDFLAG(IS_COBALT)